lib/mempool.go

```diff
@@ -1451,18 +1452,28 @@ func ComputeTransactionMetadata(txn *MsgDeSoTxn, utxoView *UtxoView, blockHash *
 		realTxMeta := txn.TxnMeta.(*NFTBidMetadata)
 		_ = realTxMeta
 
+		isBuyNow := false
+		// Check if the Bid is on a Buy Now NFT.
+		if val, exists := txn.ExtraData[IsBuyNowKey]; exists && reflect.DeepEqual(val, []byte{BoolToByte(true)}) {
```

Comment:
It seems a bit fragile that TxIndex relies on the IsBuyNowKey being in the ExtraData of NFTBid's in order to determine if they are buy now bids while _connectNFTBid() relies on the IsBuyNow param on the NFT Entry.  I suggest we get rid of the extradata on the NFTBid and instead check to see if the NFTEntry that was bid on is now owned by the bidder.  Wdyt?

Reply:
seems reasonable to me!